drift_dev/lib/src/model/sql_query.dart

```diff
@@ -157,6 +158,28 @@ abstract class SqlQuery {
 
     return resultClassName;
   }
+
+  /// Returns all found elements, from this query an all nested queries. The
+  /// elements returned by this method are in no particular order, thus they
+  /// can only be used to determine the method parameters.
+  ///
+  /// This method makes some effort to remove duplicated parameters. But only
+  /// by comparing the dart name.
+  List<FoundElement> elementsWithNestedQueries() {
+    final elements = List.of(this.elements);
+
+    final subQueries = resultSet?.nestedResults.whereType<NestedResultQuery>();
+    for (final subQuery in subQueries ?? const <NestedResultQuery>[]) {
+      for (final subElement in subQuery.query.elementsWithNestedQueries()) {
+        if (elements
+            .none((e) => e.dartParameterName == subElement.dartParameterName)) {
+          elements.add(subElement);
+        }
+      }
+    }
+
+    return elements;
+  }
```

Comment:
I added support for the dart placeholder :) I mainly handled them the same way like variables. But I think this method needs some more attention, because this is used to generate code that is required for the main query and all nested queries (for example method parameters).